portingdb/templates/mispackaged.html

```diff
@@ -0,0 +1,70 @@
+{% extends "_base.html" %}
+
+{% block titlecontent -%}
+    Mispackaged – {{ super() }}
+{%- endblock titlecontent %}
+
+{% block style %}
+{{ super() }}
+
+body {
+  /*font: 10px sans-serif;*/
+}
+
+{% endblock style %}
```

Comment:
It looks like I can safely remove all that except the first line: `{% extends "_base.html" %}`.

If I remove that, I get an error: `jinja2.exceptions.UndefinedError: 'pkglink' is undefined`, because the `pkglink` macro is defined there. Thus I think the first line should stay on.